src/components/Menu/MenuItem.tsx

```diff
@@ -8,7 +8,10 @@ import {eventBroker} from '../utils/event-broker';
 const b = block('menu');
 
 export interface MenuItemProps extends DOMProps, QAProps {
+    /** @deprecated use `iconStart` instead */
```

Comment:
With this change, the `DropdownMenu`'s README also requires an update, at least in the sections *Custom icons* and *DrowdownMenuItem*

Reply:
Done